### PR TITLE
PRODEM update

### PR DESCRIPTION
**Pull request by mankoff**
This PR changes the code (and text) to use the PRODEM product (in review at https://doi.org/10.5194/essd-2023-224).

This means both the baseline DEM changes from GIMP to PRODEM, and 2019 forward dh/dt changes from Khan_2016 to PRODEM. Historical dh/dt remains Khan.

This addresses #44 and changes are documented there.

### CODE REVIEW

File: ice_discharge.org [hunk 1]
```diff
@@ -1801,6 +1778,24 @@ parallel "r.colors map=dh_{1} color=difference" ::: $(seq 1995 2019)
...
+g.mapset -c DEM
+
+g.region raster=DEM_2020@PRODEM -pa
+for y in {2019..2022}; do
+  r.mapcalc "DEM_${y} = DEM_${y}@PRODEM"
+done
+
+for y in {2019..1995}; do
+  y1=$(( ${y} + 1 ))
+  r.mapcalc "DEM_${y} = DEM_${y1} - dh_${y}@Khan_2016"
```
Comment (signehl) [inline ice_discharge.org:1796]: I don't understand why 2020 is the baseline DEM, when 2019 is the first PRODEM. With this line DEM_2019 is calculated from the Khan dh or am I missing something?

File: ice_discharge.org [hunk 2]
```diff
@@ -161,19 +160,17 @@ After updates, re-run make, and then...
...
 
-BedMachine has been updated from v3 to v4. Uncertainty has declined by ~10 % (that is, from ~10 % of total discharge to ~9 %, or from ~50 Gt yr^{-1} to ~45 Gt yr^{-1}). With the BedMachine v4 update, the southeast sector is no longer patched with citet:millan_2018 data.
+The elevation change code and data have been completely rewritten. Previous versions used the GIMP DEM (NSIDC 0645; citet:howat_2014_greenland,NSIDC_0645), a DEM generated from several years of data but with each pixel time-stamped. Elevation change was generated per-pixel using the citet:khan_2016_geodetic elevation change data set. The new workflow uses the PROMICE PRODEM 2019 DEM (CITE winstrup) as the baseline DEM, and annual PRODEMs forward in time from there. When no PRODEM exists (at the moment, for the past year) there is no computed change in elevation. Prior to 2019, we still use citet:khan_2016_geodetic to the beginning of that dataset which is 1995. Prior to 1995 there is no computed change in elevation. Annual DEMs and change in elevations are interpolated linearly and used to estimate a thickness at each velocity timestamp.
```
Comment (signehl) [inline ice_discharge.org:169]: I think there is a mismatch between the stated baseline DEM is from 2019, but in the code 2020 is used?

**Review by signehl — COMMENTED** [review]
Thanks for this update. Please see my comment about the baseline DEM. I will run a test today and compare with the version using PPROMICE IV 200m res but without the PRODEM correction.

**Comment by mankoff** [comment]
> I will run a test today and compare with the version using PPROMICE IV 200m res but without the PRODEM correction.

I ran this comparison already  - see comment on #44.

**Comment by mankoff** [comment]
Closes #44

**Pull request merged**